hashtagsv2/graphs/views.py

```diff
@@ -216,30 +230,22 @@ def projects_csv(request):
     return response
 
 def time_csv(request):
-    request_dict = request.GET.dict()
+    temp = time_statistics_data(request)
+    temp = temp.content
+    # converting byte str to str
+    temp = temp.decode('ASCII')
+    temp = json.loads(temp)
+
     response = HttpResponse(content_type='text/csv')
     response['Content-Disposition'] = 'attachment; filename="hashtags_dates.csv"'
-
-    hashtags = hashtag_queryset(request_dict)
-    earliest_date = hashtags[len(hashtags)-1].timestamp.date()
-    latest_date = hashtags.first().timestamp.date()
-
-    time_dic = {}
-    qs = hashtags.annotate(day = TruncDay('timestamp')).values('day').annotate(edits = Count('rc_id')).order_by()
-    for item in qs:
-        time_dic[item['day'].date()] = item['edits']
+    
     writer = csv.writer(response)
     writer.writerow([
         # Translators: Date on which edit is made.
         _('Date'),
```

Comment:
Just a small change and it will be good to go. CSV header should be also be changed to `month`, `date` and `year` accordingly.

Reply:
done :+1: